### ISSUE TITLE
Evaluate approach how specific location can have their dedicated description/assets

### ISSUE THREAD

**Issue by santilland** [eurodatacube/eodash-catalog#74]
Looking at indicator E9 of trilateral, each location has a dedicated markdown file.
In principle i think we could "just" provide a new description for the item and have a generic for the collection, but we need to evaluate how we can configure this so it is injected during generation of the items.
In the E9 case it would be for geodb data.

**Comment by lubojr** [comment]
I would assume this falls under homogenization of the metadata on a per-collection level. I don't think we should have to implement this on location level?

**Comment by santilland** [comment]
Good point, as next steps we discussed cleaning the description markdown files to be data descriptive and not have "stories" within. So it is possible this would no longer be needed then.
Let's keep this issue open and collect this points to discuss them in our next "data migration" session.
